modules/ROOT/pages/instanton.adoc

```diff
@@ -338,6 +338,36 @@ When you deploy to Kubernetes, the container must be granted the `CHECKPOINT_RES
             - ALL
 ----
 
+==== Red Hat OpenShift security context constraints
+
+To deploy applications to Red Hat OpenShift with InstantOn, you must specify a security context constraint (SCC) that at a minimum grants the capabilities to the defaults that are needed for InstantOn. The following example defines an SCC with these capabilities that is called `defaultAddCapabilities`:
+
+[source,yaml]
+----
+defaultAddCapabilities:
+- CHECKPOINT_RESTORE
+- SETPCAP
+----
+
+The applications you deploy must be associated with this SCC, for example, by specifying the `serviceAccountName` setting in the deployment yaml file. The following example specifies the `defaultAddCapabilities` SCC that was defined in the previous example:
```

Comment:
This is confusing because `defaultAddCapabilities` is treated as if it is an SCC name, but before `defaultAddCapabilities` is a parameter of the SCC yaml file.  I think it would be better to use a name like `liberty-instanton-scc`:

> The applications you deploy must be associated with an SCC that adds the required capabilities. For example, assume there is an SCC deployed called `liberty-instanton-scc` that added the required capabilities.  The following example specifies the `serviceAccountName` parameter to set the SCC name to `liberty-instanton-scc`: